dbms/src/DataStreams/WindowBlockInputStream.cpp

```diff
@@ -509,13 +829,7 @@ void WindowTransformAction::advanceFrameEnd()
     }
     case WindowFrame::BoundaryType::Offset:
```

Comment:
In `WindowDescription::setWindowFrame`, for WindowBoundType of both `Following` and `Preceding` is converted to `WindowFrame::BoundaryType::Offset`, can you double confirm that if this information lost is as expected after we truely support frame?

Reply:
> In `WindowDescription::setWindowFrame`, for WindowBoundType of both `Following` and `Preceding` is converted to `WindowFrame::BoundaryType::Offset`, can you double confirm that if this information lost is as expected after we truely support frame?

According to tidb, `Following` and `Preceding` is useless when frame type is range.